packaging/Jenkinsfile

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/groovy
```

Comment:
This file doesn't need to be added.  Did you use `<path_to_local_packaging_dir>/install` to update the packaging?  IIRC, that only copied the needed files for a `packaging/` update.